src/index.html

```diff
@@ -10,23 +11,40 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar__container">
+      <form
+        class="search-bar__form"
+        action="#"
```

Comment:
The form action attribute is set to `'#'`, which means it will not submit data to a server. If this is not the intended behavior, please specify a valid URL or server endpoint.